target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -172,6 +172,9 @@ ramips_board_detect() {
 	*"E1700")
 		name="e1700"
 		;;
+	*"ESR600")
+		name="esr600"
+		;;
```

Comment:
This whole case can be removed.

Reply:
Removed this case, which means that the file no longer has any ESR600 specific changes